pkgs/by-name/lz/lz4json/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  lz4,
+  pkg-config,
+}:
+
+stdenv.mkDerivation {
+
+  pname = "lz4json";
+  version = "2-unstable-2019-12-29";
+  src = fetchFromGitHub {
+    owner = "andikleen";
+    repo = "lz4json";
+    rev = "c44c51005c505de2636cc1e59cde764490de7632";
+    sha256 = "sha256-rLjJ7qy7Tx0htW1VxrfCCqVbC6jNCr9H2vdDAfosxCA=";
+  };
+
+  buildInputs = [
+    lz4
+    pkg-config
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    install -D lz4jsoncat $out/bin/lz4jsoncat
+  '';
+
+  meta = {
+    description = "LZ4 JSON compressor for Mozilla proprietary format";
+    homepage = "https://github.com/andikleen/lz4json";
+    license = lib.licenses.unfree;
```

```suggestion
    license = lib.licenses.bsd2;
```
Compare https://github.com/andikleen/lz4json/blob/master/lz4jsoncat.c#L7-L16 and https://spdx.org/licenses/BSD-2-Clause.html

---

It is likely be BSD2, but the only part of license text is written or does not have LICENSE file which contains a full license text. So I was unsure. This really meant to be BSD2 clause.